test/org/sagebionetworks/bridge/validators/StudyParticipantValidatorTest.java

```diff
@@ -120,108 +115,95 @@ public void validPasses() {
     }
     
     @Test
-    public void emailRequired() {
+    public void emailOrPhoneRequired() {
         validator = new StudyParticipantValidator(study, true);
-        assertCorrectMessage(withEmail(null), "email", "email is required");
+        assertValidatorMessage(validator, withEmail(null), "StudyParticipant", "email or phone is required");
     }
     
     @Test
     public void passwordRequired() {
         validator = new StudyParticipantValidator(study, true);
-        assertCorrectMessage(withPassword(""), "password", "password is required");
+        assertValidatorMessage(validator, withPassword(""), "password", "is required");
     }
```

Comment:
Unit test for null password okay?

Reply:
Yes it's now different from this case. Added an additional test.